modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,55 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-config_{context}"]
+= Configuration parameters for shared VPC installation
+There are several important configuration parameters which are required to install {product-title} on GCP using a shared VPC. The following is a sample `install-config.yaml` file which demonstrates these fields.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+credentialsMode: Passthrough <1>
+metadata:
+  name: cluster_name
+platform:
+  gcp:
+    computeSubnet: shared-vpc-subnet-1 <2>
+    controlPlaneSubnet: shared-vpc-subnet-2 <3>
+    createFirewallRules: Disabled <4>
+    network: shared-vpc <5>
+    networkProjectID: host-project-name <6>
+    publicDNSZone:
+      id: public-dns-zone <7>
+      project: host-project-name <8> 
+    projectID: service-project-name <9>
+    region: us-east1
+    tags: <10>
+      - global-tag1
+compute:
+  platform:
+    gcp:
+      tags: <10>
+        - compute-tag1
+controlPlane:
+  platform:
+    gcp:
+      tags: <10>
+        - control-plane-tag1
+----
+<1> `credentialsMode` must be set to Passthrough to allow the cluster to use the provided GCP service account after cluster creation.
+<2> The name of the subnet in the shared VPC for compute machines to use.
+<3> The name of the subnet in the shared VPC for control plane machines to use.
+<4> Optional. If you set `createFirewallRules` to "Disabled", you can create and manage firewall rules manually through the use of network tags. By default, the cluster will automatically create and manage the firewall rules that are required for cluster communication. Your service account must have `compute.NetworkAdmin` and `compute.securityAdmin` privileges in the host project to perform these tasks automatically. 
+<5> The name of the shared VPC.
+<6> The name of the host project where the shared VPC exists.
+<7> Optional. The name of a public DNS zone in the host project. Your service account must have the `dns.admin` privilege in the host project. If you do not set this value, the installation program will use the public DNS zone in the service project.
+<8> Optional. The name of the host project which contains the public DNS zone. This value is required if you set `platform.gcp.publicDNSZone.id`.
+<9> The name of the GCP project where you want to install the cluster.
+<10> Optional. If you want to manually create and manage your GCP firewall rules, you must set `platform.gcp.createFirewallRules` to "Disabled" and then specify one or more network tags. You can set tags to apply to the compute machines, the control plane machines, or all machines.
```

Comment:
Using tags isn't technically required for manually managed firewall rules. It is possible to use cluster-agnostic rules that open up all of the right ports without the use of any tags. However, it is highly recommended that the tags be used to isolate the traffic per cluster, mimicking the default installer behavior.